src/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ include_directories(3rd_party)
 include_directories(3rd_party/SQLiteCpp/include)
 include_directories(3rd_party/sentencepiece)
 include_directories(3rd_party/fbgemm/include)
+include_directories(3rd_party/intgemm)
+include_directories(${CMAKE_BINARY_DIR}/src/3rd_party/intgemm)
```

Comment:
No, one of them refers to the source directory and the other one refers to the build directory. This is necessary because we have configuration generated in the build directory.

Reply:
We do compilation tests to check if the compiler supports AVX512BW and AVX512VNNI.  That generates a header ${CMAKE_BINARY_DIR}/src/3rd_party/intgemm/intgemm_config.h .  

intgemm itself has relative paths.  So if Marian included intgemm by relative path, we could remove `include_directories(3rd_party/intgemm)`.

Reply:
Whichever the path will be, could you add a comment why it's needed as explained in the response above?